packages/angular_devkit/build_angular/BUILD.bazel

```diff
@@ -318,7 +318,7 @@ LARGE_SPECS = {
     },
     "extract-i18n": {},
     "karma": {
-        "shards": 3,
+        "shards": 6,
```

Comment:
Theory: We're doubling the number of tests, so we can (and should) double the number of shards.